Misc doc updates (#18572)
unoplatform/uno: doc/articles/migrating-from-previous-releases.md

NETSDK1198: A publish profile with the name 'win-AnyCPU.pubxml' was not found in the project.
```

> This issue is not yet fixed but does not cause any issues while deploying.

Reviewer: ```suggestion
This issue is not yet fixed, but it does not cause any problems during deployment.
```
